.github/workflows/e2e_tests.yml

```diff
@@ -43,6 +43,10 @@ jobs:
             composer install
             npm ci
             npm run dist
+      - name: Install docker-compose
+        run: |
+          sudo curl -L "https://github.com/docker/compose/releases/download/v2.3.3/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
+          sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
```suggestion
          sudo chmod 550 /usr/local/bin/docker-compose
```

Reply:
The change, although more secure, wouldn't change much in this context.

Reply:
@Ferdev The change made the E2E setup fail, see https://github.com/Automattic/woocommerce-services/actions/runs/10366467965/job/28695765567, so I reverted it back.

Reply:
sounds good! Pretty pretty weird that GitHub is running `docker-composer` with a user distinct than the owner, or not even adding the user they're using to the owner's group